funding/manager.go

```diff
@@ -100,6 +100,11 @@ const (
 	// for the funding transaction to be confirmed before forgetting
 	// channels that aren't initiated by us. 2016 blocks is ~2 weeks.
 	maxWaitNumBlocksFundingConf = 2016
+
+	// checkPeerFundingLockInterval is used when we are waiting for the
+	// peer to send us FundingLocked. We will check every 10ms to see if
+	// the message is received.
+	checkPeerFundingLockInterval = 10 * time.Millisecond
```

Comment:
Alternatively, can we keep the longer timeout (1 second), and then make this a build/config flag? 

10 ms still seems kinda short, and can end up making a toon of log lines.

Reply:
Used a build flag instead. Was thinking about making it a config but then think maybe it's better to not expose this value. Also I think we could remove this interval once `receivedFundingLocked` can be blocking until funding locked is received and processed.